pkg/api/bytes.go

```diff
@@ -66,7 +66,7 @@ func (s *server) splitUpload(ctx context.Context, r io.ReadCloser, l int64) (int
 		response.Message = "upload error"
 		response.Code = http.StatusInternalServerError
 		err = fmt.Errorf("%s: %v", response.Message, err)
-		return response, err
+		return swarm.ZeroAddress, err
```

Comment:
The `response` variable is not used, it can be removed, also the fmt.Errorf.

Reply:
moved this up to the `bytesUploadHandler` for now